### ISSUE TITLE
[Proposed] Understanding ECMA

### ISSUE THREAD

**Issue by dshaw** [dshaw/alwaysforward#16]
## Title

Always Forward of dshaw: Understanding ECMA

## Topic

TBD

## Participant(s)

* Dan Shaw - @dshaw - Strategic Advisor

CANDIDATES:
* 

## Schedule

TBD

## Agenda

No agenda

## Transcript

Not available

## Link(s)

* 

## Meta

#dshawaf #AlwaysForward #nodejs

**Comment by bterlson** [comment]
Sure, happy to help, though I see lots of awesome candidates already 😊

**Comment by dshaw** [comment]
Yes and yes. We really have an embarrassing number of qualified individuals. Thank you all.

**Comment by littledan** [comment]
Ditto what @bterlson said.

**Comment by jorydotcom** [comment]
Double ditto @littledan @bterlson 😄

**Comment by dshaw** [comment]
Show link is up and ready: https://www.crowdcast.io/e/dshawaf7

![dshawaf7-cover](https://user-images.githubusercontent.com/4322/43483270-fbd69e00-94bf-11e8-9edb-513e5e08a89c.png)

**Comment by jorydotcom** [comment]
I just really want to appreciate @tzmanics 's photoshop skills here, this looks like a high-drama movie poster from the 90s and I am dying 😂
